### PR TITLE
History of Cosmos Hub Table

### PR DESCRIPTION
**Pull request by okwme**
Started writing this up because I wanted to provide some instructions for how to run historical nodes to make historical queries and realized you need a lot of context to do it correctly. I started collecting all the information I thought was relevant to the various networks that have been run as part of the continuous cosmos hub and put them in this PR. This might not be the best location for this information but I think it's valuable to collect and we can discuss here where the best use of it might live.

### CODE REVIEW

File: docs/roadmap/README.md [hunk 1]
```diff
@@ -6,4 +6,22 @@ parent:
...
+|---------------------|---------------|-----------|---------------|------------|------------|--------------------------|--------------------------|
+| Mainnet Launch      | ?             | 0         | `cosmoshub-1` | ?          | v0.31.x    | _Included in Cosmos SDK_ | n/a                      |
+| Upgrade #1          | ?             | 500,043   | `cosmoshub-2` | ?          | v0.36.x    | v1.0.x                   | n/a                      |
+| Upgrade #2          | Dec 2019?     | 2,902,000 | `cosmoshub-3` | ?          | v0.37.x    | v2.0.x                   | n/a                      |
+| Stargate            | ?             | 5,200,791 | `cosmoshub-4` | ?          | v0.40.x    | v3.0.x                   | _Included in Cosmos SDK_ |
+| Security Hard Fork  | ?             | ?         | `cosmoshub-4` | ?          | v0.41.x    | v4.0.x                   | _Included in Cosmos SDK_ |
+| Delta (Gravity DEX) | ?             | 6,910,000 | `cosmoshub-4` | ?          | v0.42.x    | v5.0.x                   | _Included in Cosmos SDK_ |
+| Vega                | Dec 13, 2021? | 8,695,000 | `cosmoshub-4` | ?          | v0.44.x    | v6.0.x                   | v2.0.0                   |
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
| Vega                | 2021-12-13 | 8,695,000 | `cosmoshub-4` | ?          | v0.44.x    | v6.0.x                   | v2.0.0                   |
```

File: docs/roadmap/README.md [hunk 2]
```diff
@@ -6,4 +6,22 @@ parent:
...
+
+For current roadmap, check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+
+## Cosmos Hub History
+
+| Upgrade Name        | Date          | Height    | Chain ID      | Tendermint | Cosmos SDK | Gaia                     | IBC                      |
+|---------------------|---------------|-----------|---------------|------------|------------|--------------------------|--------------------------|
+| Mainnet Launch      | ?             | 0         | `cosmoshub-1` | ?          | v0.31.x    | _Included in Cosmos SDK_ | n/a                      |
+| Upgrade #1          | ?             | 500,043   | `cosmoshub-2` | ?          | v0.36.x    | v1.0.x                   | n/a                      |
+| Upgrade #2          | Dec 2019?     | 2,902,000 | `cosmoshub-3` | ?          | v0.37.x    | v2.0.x                   | n/a                      |
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
| Upgrade #2          | ?     | 2,902,000 | `cosmoshub-3` | ?          | v0.37.x    | v2.0.x                   | n/a                      |
```

File: docs/roadmap/README.md [hunk 3]
```diff
@@ -6,4 +6,22 @@ parent:
...
 # Roadmap
 
-Check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+## Roadmap 2.0
+
+For current roadmap, check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+
+## Cosmos Hub History
+
+| Upgrade Name        | Date          | Height    | Chain ID      | Tendermint | Cosmos SDK | Gaia                     | IBC                      |
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
| Upgrade Name        | Date          | Height    | Chain ID      | Tm | Cosmos SDK | Gaia                     | IBC                      |
```

File: docs/roadmap/README.md [hunk 4]
```diff
@@ -6,4 +6,22 @@ parent:
 
 # Roadmap
 
-Check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+## Roadmap 2.0
+
+For current roadmap, check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
For current roadmap, check out the Cosmos 2.0 [Roadmap](./cosmos-hub-roadmap-2.0.md).
```

File: docs/roadmap/README.md [hunk 5]
```diff
@@ -6,4 +6,22 @@ parent:
...
 # Roadmap
 
-Check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+## Roadmap 2.0
+
+For current roadmap, check out the Cosmos 2.0 [Roadmap](./cosmos-hub-roadmap-2.0.md).
+
+## Cosmos Hub History
+
+| Upgrade Name        | Date          | Height    | Chain ID      | Tm | Cosmos SDK | Gaia                     | IBC                      |
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
| Upgrade Name        | Date          | Height    |          Chain ID               | Tm | Cosmos SDK | Gaia                     | IBC                      |
```

File: docs/roadmap/README.md [hunk 6]
```diff
@@ -6,4 +6,22 @@ parent:
...
 # Roadmap
 
-Check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+## Roadmap 2.0
+
+For current roadmap, check out the Cosmos 2.0 [Roadmap](./cosmos-hub-roadmap-2.0.md).
+
+## Cosmos Hub History
+
+| Upgrade Name        | Date          | Height    |          Chain ID               | Tm | Cosmos SDK | Gaia                     | IBC                      |
```
Comment (okwme) [inline docs/roadmap/README.md]:
```suggestion
| Upgrade Name        | Date          | Height    | Chain Identifier | Tm | Cosmos SDK | Gaia                     | IBC                      |
```

**Comment by okwme** [comment]
https://explorer.provenance.io/stats/versions

File: docs/roadmap/README.md [hunk 7]
```diff
@@ -6,4 +6,22 @@ parent:
...
 # Roadmap
 
-Check out the Cosmos 2.0 Roadmap [roadmap](./cosmos-hub-roadmap-2.0.md).
+## Roadmap 2.0
+
+For current roadmap, check out the Cosmos 2.0 [Roadmap](./cosmos-hub-roadmap-2.0.md).
+
+## Cosmos Hub History
+
+| Upgrade Name        | Date          | Height    | Chain Identifier | Tm | Cosmos SDK | Gaia                     | IBC                      |
```
Comment (yaruwangway) [inline docs/roadmap/README.md]: Hi, time is specified to date ? hours ?

**Pull request merged**